pkg/apis/core/v1/conversion.go

```diff
@@ -540,3 +541,12 @@ func Convert_core_PersistentVolumeSpec_To_v1_PersistentVolumeSpec(in *core.Persi
 func Convert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in *v1.PersistentVolumeSpec, out *core.PersistentVolumeSpec, s conversion.Scope) error {
 	return autoConvert_v1_PersistentVolumeSpec_To_core_PersistentVolumeSpec(in, out, s)
 }
+
+func Convert_Slice_string_To_Pointer_v1_LogStreamType(in *[]string, out **v1.LogStreamType, s conversion.Scope) error {
+	if len(*in) == 0 {
```

Comment:
Was this not auto generated?

Reply:
I am not sure, if I don't provide this conversion function, a `FIXME` comment would be generated inside the `autoConvert_url_Values_To_v1_PodLogOptions` function, I guess the generator didn't know how to perform conversion between `[]string` and `*v1.LogStreamType`

Reply:
Ok.